lours/dataset/dataset.py

```diff
@@ -1006,7 +1006,12 @@ def init_annotations(self):
                 f" their string equivalent : {missing_ids}"
             )
             for i in missing_ids:
-                self.label_map[i] = str(i)
```

Comment:
Do you know why Codecov is saying that this is not covered by the tests? If it was covered before, I don't see how this new change is not